Makefile

```diff
@@ -0,0 +1,78 @@
+
+install-deps:
+	apt-get update -y
+	apt-get install --no-install-recommends -y build-essential clang protobuf-compiler ocl-icd-opencl-dev aria2 cmake
```

Comment:
These dependencies should be updated.

Reply:
These are the same ones we use in Forest. Which of them should be updated?

Reply:
Are all of them required by these crates?

Reply:
removed them